cmd/operator/main.go

```diff
@@ -114,6 +127,13 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
+	log := logrus.New()
+	failOnError := func(err error, msg string, args ...interface{}) {
```

Comment:
We can probably avoid using function for only one use case and write the code directly